.github/workflows/tests.yml

```diff
@@ -176,23 +184,23 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - sytest-tag: py35
+          - sytest-tag: stretch
```

Comment:
This is mainly a comment on sytest itself, but with `stretch`/`buster`/etc. tags its hard to parse what kind of version is being tested exactly, could you maybe add comments which map these?

```
# stretch: python 3.6
# buster: python 3.7
# testing: python 3.8
```
this, at the very least, keeps some maintainability on these tags

Reply:
See https://github.com/matrix-org/sytest/issues/1003 for context

Reply:
(I've backed this out for the moment, as we haven't actually rebuilt and retagged images yet)